gamemodes/horde/gamemode/gui/cl_leaderboard.lua

```diff
@@ -97,11 +97,15 @@ function PANEL:Paint()
 
 end
 
+local showLeaderConvar = GetConVar("horde_show_leaderboard")
+
 function HORDE:ShowLeaderboardThenFadeOut()
     HORDE.leader_board:SetVisible(true)
-    timer.Simple(10, function ()
-        HORDE.leader_board:SetVisible(false)
-    end)
+    if showLeaderConvar:GetBool() == false then 
```

Comment:
`if showLeaderConvar:GetBool() == false then` should be simplified to `if not showLeaderConvar:GetBool() then`